src/mesh/mesh.cpp

```diff
@@ -79,13 +79,25 @@ Mesh::Mesh(ParameterInput *pin, ApplicationInput *app_in, Packages_t &packages,
                  pin->GetInteger("parthenon/mesh", "nx3")},
                 {false, pin->GetInteger("parthenon/mesh", "nx2") == 1,
                  pin->GetInteger("parthenon/mesh", "nx3") == 1}),
-      mesh_bcs{
-          GetBoundaryFlag(pin->GetOrAddString("parthenon/mesh", "ix1_bc", "reflecting")),
-          GetBoundaryFlag(pin->GetOrAddString("parthenon/mesh", "ox1_bc", "reflecting")),
-          GetBoundaryFlag(pin->GetOrAddString("parthenon/mesh", "ix2_bc", "reflecting")),
-          GetBoundaryFlag(pin->GetOrAddString("parthenon/mesh", "ox2_bc", "reflecting")),
-          GetBoundaryFlag(pin->GetOrAddString("parthenon/mesh", "ix3_bc", "reflecting")),
-          GetBoundaryFlag(pin->GetOrAddString("parthenon/mesh", "ox3_bc", "reflecting"))},
+      mesh_bc_names{pin->GetOrAddString("parthenon/mesh", "ix1_bc", "reflecting"),
+                    pin->GetOrAddString("parthenon/mesh", "ox1_bc", "reflecting"),
+                    pin->GetOrAddString("parthenon/mesh", "ix2_bc", "reflecting"),
+                    pin->GetOrAddString("parthenon/mesh", "ox2_bc", "reflecting"),
+                    pin->GetOrAddString("parthenon/mesh", "ix3_bc", "reflecting"),
+                    pin->GetOrAddString("parthenon/mesh", "ox3_bc", "reflecting")},
+      // TODO(JMM): This probably needs a per-package treatment too
```

Comment:
What exactly do you mean? Are you referring to the additional per-package boundary conditions?
And should this be addressed in this PR?

Reply:
So Luke's PR #968 makes boundary conditions per-package. But I'd like to hook that into input parsing at some point. That's what this TODO is for.

Reply:
Maybe keep track as an issue then, too (on top of the `TODO` in the code)?

Reply:
:+1: #994